### PR TITLE
Add default loading wheel

### PR DESCRIPTION
**Pull request by gibbok**
Includes:
- Add a default loading wheel when application loads.
- Adjust Title tag based on `appName`

The loading wheel:

Screen Shot 2020-07-10 at 2 11 44 PM

### CODE REVIEW

File: src/templates/ui/index.ts [hunk 1]
```diff
@@ -13,15 +13,69 @@
...
+          border-right: 1.1em solid rgba(128, 128, 128, 0.2);
+          border-bottom: 1.1em solid rgba(128, 128, 128, 0.2);
+          border-left: 1.1em solid #C0C0C0;
+          -webkit-transform: translateZ(0);
+          -ms-transform: translateZ(0);
+          transform: translateZ(0);
+          -webkit-animation: load8 1s infinite linear;
+          animation: load8 1s infinite linear;
+      }
+      @-webkit-keyframes load8 {
```
Comment (Alexander89) [inline src/templates/ui/index.ts:51]: why is it called load8 ?

File: src/templates/ui/index.ts [hunk 2]
```diff
@@ -13,15 +13,69 @@
...
  * limitations under the License.
  */
-export const defaultHtml = `<!DOCTYPE html>
+export const defaultHtml = (appName: string): string => `<!DOCTYPE html>
 <html>
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Actyx-Pond-Demo!</title>
+    <title>${appName}</title>
```
Comment (Alexander89) [inline src/templates/ui/index.ts:21]: nice

File: src/templates/ui/index.ts [hunk 3]
```diff
@@ -13,15 +13,69 @@
...
+    <style>
+      .app {
```
Comment (Alexander89) [inline src/templates/ui/index.ts:23]: app is not used

File: src/templates/ui/index.ts (hunk 1 above)
Comment (gibbok) [reply src/templates/ui/index.ts]: thanks, added a better name

File: src/templates/ui/index.ts (hunk 3 above)
Comment (gibbok) [reply src/templates/ui/index.ts]: Thanks for noticing it! Now converted in a wrapper class so we can keep the spinning wheel in the center of the viewport :)

**Pull request merged**